datashuttle/utils/utils.py

```diff
@@ -9,11 +13,30 @@
 # --------------------------------------------------------------------------------------------------------------------
 
 
-def message_user(message: Union[str, list]) -> None:
+def log(message: str) -> None:
+    logger = logging.getLogger("fancylog_")
+    logger.info(message)
+
+
+def log_and_message(message: str, use_rich: bool = False) -> None:
+    log(message)
+    message_user(message, use_rich)
```

Comment:
`message_user` to me isn't super clear what it means. Something like `print_message_to_user` would be clearer.